src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -454,11 +453,10 @@ public List<uint256> GenerateStratisWithMiner(int blockCount)
         [Obsolete("Please use GenerateStratisWithMiner instead.")]
         public Block[] GenerateStratis(int blockCount, List<Transaction> passedTransactions = null, bool broadcast = true)
         {
-            FullNode fullNode = (this.runner as StratisBitcoinPowRunner).FullNode;
+            FullNode fullNode = ((StratisBitcoinPowRunner) this.runner).FullNode;
```

Comment:
Is it used much this method? `GenerateStratis`
Maybe we can get rid of it?

Reply:
this has 16 references, I didn't feel brave enough to delete it...

Reply:
Its a shame to refactor this class and not remove this method.
So keep it in mind for a next PR.

Reply:
@dangershony Im busy with a PR that removes this method, will submit PR this morning ;)